shared/templates/rsyslog_logfiles_attributes_modify/oval.template

```diff
@@ -90,27 +90,39 @@
   <!-- Define OVAL variable to hold all the various system log files locations
        retrieved from the different rsyslog configuration files
   -->
-  <local_variable id="var_rfg_log_files_paths" datatype="string" version="1" comment="File paths of all rsyslog configuration files">
-    <object_component item_field="subexpression" object_ref="object_rfg_log_files_paths" />
+  <local_variable id="var_rfg_{{{ _RULE_ID }}}_log_files_paths" datatype="string" version="1" comment="File paths of all rsyslog configuration files">
+    <object_component item_field="subexpression" object_ref="object_rfg_{{{ _RULE_ID }}}_log_files_paths" />
   </local_variable>
 
   <!-- Perform the test if all rsyslog system log files are owned by the appropriate group -->
-  <unix:file_test check="all" check_existence="all_exist" id="test_rsyslog_files_groupownership" version="1" comment="System log files are owned by the appropriate group">
-    <unix:object object_ref="object_rsyslog_files_groupownership" />
-    <unix:state state_ref="state_rsyslog_files_groupownership" />
+  <unix:file_test check="all" check_existence="all_exist" id="test_rsyslog_files_{{{ _RULE_ID }}}_ownership" version="1" comment="System log files are owned by the appropriate group">
+    <unix:object object_ref="object_rsyslog_files_{{{ _RULE_ID }}}_ownership" />
```

Comment:
Suggestion: `object_ref="object_{{{ _RULE_ID }}}"` here and on line 103.

Reply:
Thanks :bow: done in [ba94e4c](https://github.com/ComplianceAsCode/content/pull/9789/commits/ba94e4ceee13613c8fc2527e66d9ea265ddc5690)